src/include/stir/DynamicDiscretisedDensity.h

```diff
@@ -143,6 +145,9 @@ class DynamicDiscretisedDensity: public ExamData
   const singleDiscDensT & 
     get_density(const unsigned int frame_num) const ;
 
+  shared_ptr<DiscretisedDensity<3, float> >
```

Comment:
I think would need to return ` shared_ptr<const singleDiscDensT>` : correct template but also prevents modifying the underlying object.